### DISCUSSION TITLE
Further enhance the security of sensitive files

### DISCUSSION THREAD

**Discussion by zhenrong-wang** [zhenrong-wang/hpc-now#24] [Announcements]
As you may know, the connection between the local `hpcopr` environment with the cloud clusters are based on SSH. It means that `hpcopr` maintains SSH private keys. The keys are in the **SSH_KEY** directory as below:

- Windows: C:\ProgramData\hpc-now\now-ssh\
- GNU/Linux: /usr/.hpc-now/.now-ssh/
- macOS: /Applications/.hpc-now/.now-ssh/

Previously, the private keys are just hidden, not encrypted. Now, with the recently-introduced **now-crypto** AES module, we are able to encrypt/decrypt them properly.

Please stay tuned.

**Comment by pandasd** [comment]
Is there docs about the architecture?

**Reply by zhenrong-wang** [reply]
Please refer to the Readme. Currently it is the most detailed doc.
